src/utils/InputHandler.jsx

```diff
@@ -0,0 +1,24 @@
+// @flow
+
+export const inputHandler = (callback: (string, allInputTypes) => void) => (
+	event: SyntheticEvent<HTMLInputElement>
+) => {
+	if (
+		event.currentTarget != null &&
+		event.currentTarget.name != null &&
+		(event.currentTarget.value != null || event.currentTarget.checked != null)
+	) {
+		const name = event.currentTarget.name;
+		switch (event.currentTarget.type) {
```

Comment:
Default case?

Reply:
If no html input type matches, then the event is somehow invalid or with an unsupported type. In this case the callback is not called which is the desired behavior.